pkg/proxy/request.go

```diff
@@ -110,6 +118,21 @@ func getFileFromHTTP(r *http.Request, file string) (string, error) {
 	return stringTestData, nil
 }
 
+func getOverwrite(r *http.Request) (bool, error) {
+	o := r.FormValue(overwrite)
+
+	if o == "" {
+		return false, nil
+	}
+
+	overwrite, err := strconv.ParseBool(o)
+	if err != nil {
+		return false, err
+	}
+
+	return overwrite, nil
```

Comment:
```suggestion
	return strconv.ParseBool(o)
```

Reply:
This is a designed way. Because if the overwrite option is empty, `strconv.ParseBool` gives error but it should be `false` so I've implemented the error handling here.

Reply:
ohh, got it, sorry